clang/lib/Basic/OpenMPKinds.cpp

```diff
@@ -760,6 +760,9 @@ void clang::getOpenMPCaptureRegions(
     case OMPD_parallel:
       CaptureRegions.push_back(OMPD_parallel);
       break;
+    case OMPD_assume:
+      CaptureRegions.push_back(OMPD_assume);
```

Comment:
assume ends up with a simple assume intrinsic, right? Most probably non need to emit captured function, so it should be OMPD_unknown